test/test/components/previews/preview_component_preview.rb

```diff
@@ -2,7 +2,7 @@
 
 class PreviewComponentPreview < ActionView::Component::Preview
   def default
```

Comment:
With my suggestion above couldn't we make this

```ruby
def default(cta: "Click me!", title: "Lorem Ipsum")
```

that way we don't have this magic params property to know about.